chain/network/src/types.rs

```diff
@@ -813,9 +817,9 @@ pub enum NetworkRequests {
     /// (Unit tests) Fetch current routing table.
     FetchRoutingTable,
     /// Data to sync routing table from active peer.
-    Sync {
+    SyncRoutingTable {
```

Comment:
so we have enum value and struct with the same name? that might be a little bit confusing

Reply:
No, the name is different
```
pub enum PeerMessage {
    RoutingTableSync(SyncRoutingTable),
}
```

Reply:
@mm-near I renamed to the struct to be called `RoutingTableUpdate`.